src/components/layout/organize/SingleCampaignLayout.tsx

```diff
@@ -28,8 +29,13 @@ const SingleCampaignLayout: FunctionComponent<SingleCampaignLayoutProps> = ({ ch
         endDate = getNaiveDate(lastEvent.end_time);
     }
 
+    if (!campaign) return null;
```

Comment:
This by the way is a conclusion I came to after watching several of Sandi Metz videos where she uses the indentation to indicate complexity. She doesn't teach this particular principle, so I don't want to attribute the idea to her (because she may or may not agree) but [a talk of hers from 2019](https://www.youtube.com/watch?v=YtROlyWWhV0) (and other videos like it) did inspire me to this conclusion.

![image](https://user-images.githubusercontent.com/550212/129854157-7692a707-a3fb-439d-9bf1-d5106eb20abb.png)

Reply:
In all honesty I just like it. It's a newer language feature that I think improves the language for functional programming. If I am doing a one liner after a condition it's almost always a return, and I think it's a nice way to handle returns for conditions like that. I guess I just don't feel like I need the code blocks to be aware of the complexity and have gotten used to reading it.

I agree about the console log for sure though. There's also a new feature where you can automatically return from an anon function like `() => ( ... )` which is also harder to debug, esp if that `()` is over multiple lines, it's just tedious.
 
But overall I think the language is changing, and personally I like a lot of the new features.

Reply:
I don't believe single-statement blocks is a new feature, but has existed since the dawn of javascript. It's in every c-style programming language (and the debate exists in every language). 😊 

But this is something we can discuss at a separate time and agree on what to do (and potentially add it to the linter).

Reply:
Really I didn't realise I thought it was new!